assignments/4. operators/conditional-operator.md

```diff
@@ -9,22 +9,119 @@
     * [ ] Mul
     * [ ] Div
 
+  ``` js 
+  var num1 = prompt("Enter First Number");
+  num1 = Number(num1);
+  var num2 = prompt("Enter Second Number");
+  num2 = Number(num2);
+  var operation = prompt("What kind of operation you want to perform; enter - Add , Sub , Mul , Div ");
+
+  var num1 = prompt("Enter First Number");
+  num1 = Number(num1);
+  var num2 = prompt("Enter Second Number");
+  num2 = Number(num2);
+  var operation = prompt("What kind of operation you want to perform; enter - Add , Sub , Mul , Div ");
+
+  switch(operation)
+    {
+      case "Add" : function Add() {
+                    alert(num1+num2);
+                   }
+                   Add();
```

Comment:
The function name should not start with capital letters. Change the function names.